src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1094,25 +1094,33 @@ a.test-arrow:hover{
 	margin-right: 5px;
 }
 
-.sub-variant, .sub-variant > h3 {
-	margin-top: 0px !important;
-	padding-top: 1px;
+h3.variant {
+	font-weight: 600;
+	font-size: 1.1em;
+	margin-bottom: 10px;
+	border-bottom: none;
 }
 
-#main .sub-variant > h3 {
-	font-size: 15px;
-	margin-left: 25px;
-	margin-bottom: 5px;
+.sub-variant h4 {
+	font-size: 1em;
+	font-weight: 400;
+	border-bottom: none;
+	margin-top: 0;
+	margin-bottom: 0;
 }
 
-.sub-variant > div {
-	margin-left: 20px;
-	margin-bottom: 10px;
+.sub-variant {
+	margin-left: 24px;
+	margin-bottom: 20px;
 }
 
-.sub-variant > div > span {
-	display: block;
-	position: relative;
+.sub-variant {
+	margin-top: 0px !important;
+	padding-top: 1px;
```

Comment:
This is pre-existing, but maybe we should change this to 0px or 5px? It seems like an odd size.

Or control it from the `h3.variant` margin-bottom instead. It's a bit confusing to have both.